liquidctl/cli.py

```diff
@@ -72,6 +72,11 @@
 from traceback import format_exception
 
 from docopt import docopt
+try:
+    import colorlog
+    use_color_log=True
+except ImportError:
+    use_color_log=False
```

Comment:
I think it can be a normal/required dependency.

Reply:
okay, I wasn't sure how you wanted to handle it I remember seeing (or I thought I saw) something elsewhere that made use of optional dependancies for non core libraries.

Reply:
Hm, now that you mention it, do we currently have an optional dependency?

I only though about `smbus`: it was optional for a while, but was made into a conditional (Linux-only) dependency for 1.5.0.  But that was a core dependency, so you were probably thinking of something else... that I could be missing.

Reply:
I think I may just have been thinking of of the fallback usb dependancies here:
https://github.com/liquidctl/liquidctl/blob/8ef2e79fd5f385487ff4a1a8e4ee92f0670a6d7f/liquidctl/driver/usb.py#L62-L68